util/cron/test-slurm-gasnet-ibv.numa.bash

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env bash
 #
 # Multi-node, multi-locale testing on a linux64 cluster with slurm-gasnetrun_ibv launcher:
-# test numa locale model with comm=gasnet on multilocale and examples.
+# test numa locale model with comm=gasnet against "examples"
 
 CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
 source $CWD/common-slurm-gasnet-ibv.bash
-source $CWD/common-numa.bash
```

Comment:
I can see getting rid of the oversubscription variable, but I'm not sure about QT_AFFINITY.  @ronawho , was that there specifically to make oversubscribed testing work better?